speechbrain/decoders/ctc.py

```diff
@@ -965,6 +965,16 @@ def decode_beams(
                 "During decoding, going to truncate the log_probs vocab dim to match vocab_list."
             )
 
+        # check if we have log_probs
+        if not torch.allclose(
```

Comment:
This operation is called for every call, this might be a problem for people that have speed requirements.

Reply:
Yes I dunno. I can remove it if you think it is up to the user to make sure that log probs are passed in input, otherwise, I can add an input flag that is by default True, and can be switch to False if one wants to remove this check. wdyt ?

Reply:
Yes remove it, and add info in the doc about the requirements of the input